src/locales/pt/translation.json

```diff
@@ -0,0 +1,96 @@
+{
+  "channel": {
+    "beginningOf": "Inicio do #{{channel}}",
+    "file": {
+      "download": "Baixar",
+      "previewLoading": "$t(carregando)...",
+      "open": "Abrir",
+      "unableToDisplay": "Não é possível abrir o arquivo",
+      "dropzone": {
+        "add": "Adicionar arquivos ao #{{channel}"
+      }
+    },
+    "loadingHistory": "Histórico de carregamento...",
+    "loadMore": "Clique para carregar mensagens antigas",
+    "sendMessagePlaceholder": "Digite uma mensagem",
+    "status": {
+      "users": "Usário",
+      "users_plural": "Usários"
+    }
+  },
+  "configuration": "Configuração",
+  "controlPanel": {
+    "channels": "Canals",
+    "closeChannel": "Fechar",
+    "startJsIpfs": "Ligar o js-ipfs"
+  },
+  "disclaimer": {
+    "alpha": {
+      "content": "Isso é uma versao alfa do software. As coisas ao quebrar e mudar o que \u2014 As conversas serao perdidas."
+    }
+  },
+  "joinChannel": {
+    "inputPlaceholder": "Junte-se ao canal"
+  },
+  "login": {
+    "nickname": "Apelido",
+    "orLoginWith": "Ou entrar como",
+    "pressEnterToLogin": "Pressione ENTER para entrar"
+  },
+  "loading": "Carregando",
+  "settings": {
+    "names": {
+      "themeName": "Tema",
+      "language": "Idioma",
+      "sidePanelPosition": "Posição do painel lateral",
+      "colorifyUsernames": "Colorir apelidos",
+      "useEmojis": "Usar emojis",
+      "emojiSet": "Tema de Emoji",
+      "useMonospaceFont": "Usar fonte monospace",
+      "useLargeMessage": "Usar mensages grande"
+    },
+    "DESCRiptions": {},
+    "options": {
+      "themeName": {
+        "default": "Defeito",
+        "purple1": "Roxo 1",
+        "purple2": "Roxo 2",
+        "magenta": "Magenta",
+        "red": "Vermelho",
+        "orange": "Laranja",
+        "brown": "Marrom",
+        "lime": "Limão",
+        "green": "Verde",
+        "cyan": "Ciano",
+        "blue1": "Azul 1",
+        "blue2": "Azul 2",
+        "grey": "Cinzento"
+      },
+      "language": {
+        "en": "Ingles",
+        "fi": "Finlandês"
```

Comment:
There still needs to be the translation for the language itself and Portuguese in Finnish and English translations. Otherwise, +1

Reply:
Done! Let me know if the finnish makes sense.

Reply:
It indeed does :)